regular/coc.html

```diff
@@ -0,0 +1,160 @@
+<p>Additionally, we have a few IRC channels: #hacksoc is bridged to #general, #hacksoc-bottest is linked to #bot-testing, and #hacksoc-committee (a locked channel) is bridged to #committee. The bridge used is <a href="https://github.com/LukeMoll/slack-irc" target="_blank">slack-irc</a>. These channels are mostly used by alumni, and also contain Mathison, a bot that makes things easier for people on irc, which can be found <a href="https://github.com/HackSoc/csbot" target="blank">here</a>. </p>
+<p>
+    To make everything extra fun, different bridges bridge different things:
+<table cellspacing="10">
+    <thead>
+        <tr>
+            <th></th>
+            <th>slack-discord-bridge</th>
+            <th>sosig</th>
+            <th>slack-irc</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>joins, leaves</td>
+            <td>yes</td>
+            <td>yes</td>
+            <td>no</td>
+        </tr>
+        <tr>
+            <td>bots (e.g. slackbot)</td>
+            <td>yes</td>
+            <td>yes</td>
+            <td>no</td>
+        </tr>
+        <tr>
+            <td>threads</td>
+            <td>yes</td>
+            <td>no</td>
+            <td>yes</td>
+        </tr>
+        <tr>
+            <td>thread replies to channel</td>
+            <td>yes</td>
+            <td>yes</td>
+            <td>no</td>
+        </tr>
+    </tbody>
+</table>
+</p>
+<p>
+    Eventually, the aim is to have everything bridged using sosig. If you want to be sure something won’t be bridged elsewhere, post it in #random.
+</p>
+<h3>Content Warnings</h3>
+<p>When posting topics that might be sensitive or upsetting to some people, we like to use Slack’s threads as a type of content warning. The first message is the content warning in brackets, while the actual message would be in the thread replies. Because of the aforementioned bridge situation, this is generally done in unbridged channels like #random or #politics.</p>
+<p>
+    There is no hard rule about what topics should be CWed, but some topics you might want to consider warning for are:
+<ul>
+    <li>homophobia/transphobia/other types of queerphobia</li>
+    <li>ableism</li>
+    <li>racism</li>
+    <li>COVID-19</li>
+    <li>references to suicide or self harm</li>
+</ul>
+</p>
+<p>
+    Some topics that people might not want to deal with, like food or politics, have their own channels, and efforts should be made to keep those topics to those channels.  
+</p>
+<p>
+    It is helpful if CWs are fairly descriptive, as some people will want to know about, say, policies that might affect them personally, but not be able to deal with seeing random hate against them. for example, one might have the following CWs for transphobia
```

Comment:
suuper minor nit for consistency with the next paragraph

```suggestion
    It is helpful if CWs are fairly descriptive, as some people will want to know about, say, policies that might affect them personally, but not be able to deal with seeing random hate against them. For example, one might have the following CWs for transphobia:
```